Algorithms/GEMETD.py

```diff
@@ -0,0 +1,56 @@
+from Algorithms.ETDLB import ETDLB
+from Algorithms.BaseTD import BaseTD
```

Comment:
I think you will not need to import this anymore?

Reply:
BaseTD is still used in `learn_multiple_policies` to call BaseTD's `learn_multiple_policies` method, similar to how ETDLB calls BaseTD's `learn_multiple_policies` method. I wasn't able to find a better way of achieving this functionality (calling super()super() doesn't work) in Python.

Reply:
I don't think importing Base TD is necessary. I will take care of it.